hazelcast-jet-sql/src/test/java/com/hazelcast/jet/sql/SqlTestSupport.java

```diff
@@ -120,6 +120,21 @@ public static void assertRowsAnyOrder(String sql, Collection<Row> expectedRows)
         assertThat(actualRows).containsExactlyInAnyOrderElementsOf(expectedRows);
     }
 
+    public static void assertContainsOnlyOneOfRows(String sql, Row... expectedRows) {
```

Comment:
This variant is too niche for the utility, I'd move it to the test class. Also the `assertContainsSubsetOfRows`. I don't think it will be used in any other test than for the LIMIT clause.